src/test/common/utils/decorators.unit.test.ts

```
@@ -171,6 +173,52 @@ suite('Common Utils - Decorators', () => {
         expect(one.calls).to.deep.equal(['run']);
         expect(one.timestamps).to.have.lengthOf(one.calls.length);
     });
+    test('Debounce: multiple async call', async () => {
+        const wait = 100;
+        // tslint:disable-next-line:max-classes-per-file
+        class One extends Base {
+            @makeDebounceAsyncDecorator(wait)
+            public async run(): Promise<void> {
+                this._addCall('run');
+            }
+        }
+        const one = new One();
+
+        const start = Date.now();
+        one.run().ignoreErrors();
```

While it make sense to not await these, ignoring errors doesn't seem right either...

---

* This is a test, 
* We wrote the code for `run` and it doesn't throw exceptions
* We validate the number of times the `run` method is invoked

`ignoreErrors` is required to prevent dangling promises, i.e. ALL promises MUST be handled. You cannot invoke a method and not capture the result, that's a no-no in node.js & can potentially crash the `node.js` process if exceptions are raised.